.travis.yml

```diff
@@ -9,3 +9,8 @@ script:
   # the "yast-travis-cpp" script is included in the base yastdevel/cpp image
   # see https://github.com/yast/docker-yast-cpp/blob/master/yast-travis-cpp
   - docker run -it -e TRAVIS=1 -e TRAVIS_JOB_ID="$TRAVIS_JOB_ID" yast-ycp-ui-bindings-image yast-travis-cpp
+
+after_success:
+  - echo $TRAVIS_EVENT_TYPE
```

Comment:
yes, I plan to keep it there for some kind of verified that it always send only from our repo and master